model_analyzer/perf_analyzer/perf_config.py

```diff
@@ -95,6 +95,10 @@ def update_config(self, params=None):
             for key in params:
                 self[key] = params[key]
 
+    def deserialize(self, perf_config_dict):
```

Comment:
What about using a static `from_dict` method that returns an object using the state dictionary?

Reply:
Yes I think this makes sense.